addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6466,6 +6605,33 @@ registry.ImageTools = ImageHandlerOption.extend({
         if (params.name && params.name.startsWith("img_shape_transform_")) {
             return this._isTransformableShape() && !this._isDeviceShape();
         }
+        if (widgetName === "hover_effect_none_opt") {
+            // The hover effects are removed with the "WebsiteAnimate" animation
+            // selector so this option should not be visible.
+            return false;
+        }
+        if (params.optionsPossibleValues.setImgShapeHoverEffect) {
+            const img = this._getImg();
+            return img.classList.contains("o_animate_on_hover") && !this._isDeviceShape() && !this._isAnimatedShape();
+        }
+        // If "Description" or "Tooltip" options.
+        if (["alt", "title"].includes(params.attributeName)) {
+            const img = this._getImg();
+            return !img.matches("[data-oe-type='image'] > img, [data-oe-xpath]");
+        }
```

Comment:
This change was made to be able to put the shape option higher in the image options